django/thunderstore/api/cyberstorm/views/team.py

```diff
@@ -0,0 +1,46 @@
+from django.http import Http404, HttpRequest, HttpResponse
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from thunderstore.api.cyberstorm.serializers import (
+    CyberstormTeamMemberSerializer,
+    CyberstormTeamSerializer,
+)
+from thunderstore.repository.models.team import Team
+
+
+class TeamAPIView(APIView):
```

Comment:
This also should probably use the `RetrieveAPIView`.
See: 
- https://www.django-rest-framework.org/api-guide/generic-views/#retrieveapiview
- https://www.django-rest-framework.org/api-guide/generic-views/#genericapiview